ydb/library/yql/udfs/common/knn/knn-serializer.h

```diff
@@ -129,11 +143,11 @@ class TKnnBitVectorSerializer {
         }
     }
 
-    static const TArrayRef<const ui64> GetArray64(const TStringRef& str) {
+    static std::pair<const ui64*, ui64> GetArray(TStringRef str) {
```

Comment:
1. Please, revert to `const TStringRef& str`. It's widely used in all the other UDF functions.
2. I would rather revert to `TArrayRef` because it says "we return sized array, not just pair of values".

Reply:
1. I can revert it because any meaningful compiler will inline this function. But in general <= 16 trivially copyable bytes especially if it's single argument better to pass by value https://godbolt.org/z/aza7denTs 
2. I don't think it's good idea, because second is bit length, not `ui64 length`